packages/astro/src/config/index.ts

```diff
@@ -1,12 +1,12 @@
-import type { UserConfig } from 'vite';
+import type { UserConfig as UserViteConfig } from 'vite';
 import type { AstroInlineConfig, AstroUserConfig } from '../@types/astro.js';
 import { Logger } from '../core/logger/core.js';
 
 export function defineConfig(config: AstroUserConfig) {
 	return config;
 }
 
-export function getViteConfig(inlineConfig: UserConfig, inlineAstroConfig: AstroInlineConfig = {}) {
+export function getViteConfig(inlineUserViteConfig: UserViteConfig, inlineAstroConfig: AstroInlineConfig = {}) {
```

Comment:
nit, but could we rename this as `viteUserConfig: ViteUserConfig`? The order will also match Astro's internal type, like `AstroUserConfig` this way.

I also think that we could remove the "inline" because in Vite (and Astro), "inline" and "user" have two separate meanings. "inline" extends from the "user" interface.

Reply:
Done